buildstockbatch/gcp/gcp.py

```diff
@@ -910,21 +910,23 @@ def start_combine_results_job_on_cloud(self, results_dir, do_timeseries=True):
 
         # Create the job
         jobs_client = run_v2.JobsClient()
-        jobs_client.create_job(
+        op = jobs_client.create_job(
             run_v2.CreateJobRequest(
```

Comment:
wondering if we should also change this to `request = run_v2.CreateJobRequest(`?

Reply:
Are you just referring to the variable name? I used `op` because `create_job` [returns an Operation object](https://cloud.google.com/python/docs/reference/run/latest/google.cloud.run_v2.services.jobs.JobsClient#google_cloud_run_v2_services_jobs_JobsClient_run_job).

Reply:
My comment was more of a readability comment. when creating a job with the JobsClient in Google Cloud Run, should you pass the CreateJobRequest object using the request keyword argument? 
like 

```
op = jobs_client.create_job(
            request = run_v2.CreateJobRequest(
```

Reply:
Oh, I see. Updated!